includes/admin_import.php

```diff
@@ -4,743 +4,1212 @@
-	if (isset($_POST['tsml_add_google_maps_key']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
-		$key = sanitize_text_field($_POST['tsml_add_google_maps_key']);
-		if (empty($key)) {
+			//there can be only one
+			$tsml_google_maps_key = null;
 			delete_option('tsml_google_maps_key');
-			tsml_alert(__('API key removed.', '12-step-meeting-list'));
-		} else {
-			update_option('tsml_google_maps_key', $key);
-			$tsml_google_maps_key = $key;
-			tsml_alert(__('API key saved.', '12-step-meeting-list'));
 		}
 
-		//there can be only one
-		$tsml_mapbox_key = null;
-		delete_option('tsml_mapbox_key');
-	}
+		//add a Google API key
+		if (isset($_POST['tsml_add_google_maps_key']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
+			$key = sanitize_text_field($_POST['tsml_add_google_maps_key']);
+			if (empty($key)) {
+				delete_option('tsml_google_maps_key');
+				tsml_alert(__('API key removed.', '12-step-meeting-list'));
+			} else {
+				update_option('tsml_google_maps_key', $key);
+				$tsml_google_maps_key = $key;
+				tsml_alert(__('API key saved.', '12-step-meeting-list'));
+			}
 
-	//change geocoding method
-	if (!empty($_POST['tsml_geocoding_method']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
-		$tsml_geocoding_method = sanitize_text_field($_POST['tsml_geocoding_method']);
-		update_option('tsml_geocoding_method', $tsml_geocoding_method);
-		tsml_alert(__('Geocoding method updated.', '12-step-meeting-list'));
-	}
-?>
-	<div class="wrap">
-		<h2><?php _e('Import & Settings', '12-step-meeting-list') ?></h2>
+			//there can be only one
+			$tsml_mapbox_key = null;
+			delete_option('tsml_mapbox_key');
+		}
 
-		<div id="poststuff">
-			<div id="post-body" class="columns-2">
-				<div id="post-body-content">
+		//change geocoding method
+		if (!empty($_POST['tsml_geocoding_method']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
+			$tsml_geocoding_method = sanitize_text_field($_POST['tsml_geocoding_method']);
+			update_option('tsml_geocoding_method', $tsml_geocoding_method);
+			tsml_alert(__('Geocoding method updated.', '12-step-meeting-list'));
+		}
 
-					<?php if ($error) { ?>
-						<div class="error inline">
-							<p><?php echo $error ?></p>
-						</div>
-					<?php } elseif ($total = count($meetings)) { ?>
-						<div id="tsml_import_progress" class="progress" data-total="<?php echo $total ?>">
-							<div class="progress-bar"></div>
+		//change user interface
+		if (!empty($_POST['tsml_user_interface']) && isset($_POST['tsml_nonce']) && wp_verify_nonce($_POST['tsml_nonce'], $tsml_nonce)) {
+			$tsml_user_interface = sanitize_text_field($_POST['tsml_user_interface']);
+			update_option('tsml_user_interface', $tsml_user_interface);
+			tsml_alert(__('User Interface Display is now set to <u>' . strtoupper($tsml_user_interface) . '</u>', '12-step-meeting-list') );
```

Comment:
would be great to have this be strong instead of underline, since underline implies it's a link. also if it could read *TSML UI* instead of TSML_UI that would be excellent